java/src/test/java/ai/rapids/cudf/nvcomp/NvcompTest.java

```diff
@@ -23,18 +23,29 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.Arrays;
-import java.util.Optional;
 
 public class NvcompTest {
   private static final HostMemoryAllocator hostMemoryAllocator = DefaultHostMemoryAllocator.get();
 
   private static final Logger log = LoggerFactory.getLogger(ColumnVector.class);
 
+  private final long chunkSize = 64 * 1024;
+  private final long targetIntermediteSize = Long.MAX_VALUE;
```

Comment:
```suggestion
  private final long targetIntermediateSize = Long.MAX_VALUE;
```

Reply:
Nice catch. But I'll merge this first, and correct typo in the future PRs.